ISAAR.MSolve.SamplesConsole/Examples/Static/Linear/Hexa8SimpleExample.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Collections.Generic;
+using ISAAR.MSolve.PreProcessor;
+using ISAAR.MSolve.Solvers.Skyline;
+using ISAAR.MSolve.Problems;
+using ISAAR.MSolve.Analyzers;
+using ISAAR.MSolve.Logging;
+using ISAAR.MSolve.PreProcessor.Materials;
+using ISAAR.MSolve.PreProcessor.Elements;
+using ISAAR.MSolve.Matrices;
+
+namespace ISAAR.MSolve.SamplesConsole.Examples.Static.Linear
+{
+    class Hexa8SimpleExample
+    {
+        private static IList<Node> CreateNodes()
+        {
+            IList<Node> nodes = new List<Node>();
+            Node node1 = new Node { ID = 1, X = 0.0, Y = 0.0, Z = 0.0 };
+            Node node2 = new Node { ID = 2, X = 1.0, Y = 0.0, Z = 0.0 };
+            Node node3 = new Node { ID = 3, X = 1.0, Y = 1.0, Z = 0.0 };
+            Node node4 = new Node { ID = 4, X = 0.0, Y = 1.0, Z = 0.0 };
+            Node node5 = new Node { ID = 5, X = 0.0, Y = 0.0, Z = 1.0 };
+            Node node6 = new Node { ID = 6, X = 1.0, Y = 0.0, Z = 1.0 };
+            Node node7 = new Node { ID = 7, X = 1.0, Y = 1.0, Z = 1.0 };
+            Node node8 = new Node { ID = 8, X = 0.0, Y = 1.0, Z = 1.0 };
+            nodes.Add(node1);
+            nodes.Add(node2);
+            nodes.Add(node3);
+            nodes.Add(node4);
+            nodes.Add(node5);
+            nodes.Add(node6);
+            nodes.Add(node7);
+            nodes.Add(node8);
+            return nodes;
+        }
+
+        static void Main(string[] args)
+        {
+            VectorExtensions.AssignTotalAffinityCount();
+            double youngModulus = 200.0e06;
+            double poissonRatio = 0.3;
+            double nodalLoad = 25.0;
+
+            // Create a new elastic 3D material
+            ElasticMaterial3D material = new ElasticMaterial3D()
+            {
+                YoungModulus = youngModulus,
+                PoissonRatio = poissonRatio,
+            };
+
+            // Node creation
+            IList<Node> nodes = CreateNodes();
+
+            // Model creation
+            Model model = new Model();
+
+            // Add a single subdomain to the model
+            model.SubdomainsDictionary.Add(1, new Subdomain() { ID = 1 });
+
+            // Add nodes to the nodes dictonary of the model
+            for(int i=0; i < nodes.Count; ++i)
+            {
+                model.NodesDictionary.Add(i + 1, nodes[i]);
+            }
+
+            // Constraint bottom nodes of the model
```

Comment:
Add a for loop from 1 to 4, constraining the nodes. Too many lines for this and no need for loop unrolling!